app/assets/javascripts/discourse/app/components/composer-body.js

```diff
@@ -192,6 +192,22 @@ export default Component.extend(KeyEnterEscape, {
     );
   },
 
+  @bind
+  updateHeightOnViewportResize() {
+    const composerState = this.get("composer.composeState");
+    if (composerState !== Composer.OPEN) {
+      return;
+    }
+
+    const actualHeight = parseInt(getComputedStyle(this.element).height, 10);
+
+    this.set("composer.composerHeight", `${actualHeight}px`);
+    document.documentElement.style.setProperty(
+      "--composer-height",
```

Comment:
Since we're no longer using `composer-height`, I think we can skip adding the event listener for now.

Reply:
@pmusaraj, Although we're not using it for this, shouldn't we still keep the event listener since so that the `--composer-height` property will have the accurate height for the composer?

Reply:
It could be useful to do so, but it is best to do that if/when we need it. Also ensures that we aren't muddying the water (so to speak) for the CSS changes in this PR.

Reply:
That makes sense, I'll go ahead and remove it